app/views/casework/crime_applications/_case_details.html.erb

```diff
@@ -66,77 +66,74 @@
         <%= case_details.next_court_hearing_at %>
       </dd>
     </div>
-  </dl>
-<% end %>
-
-<% if FeatureFlags.means_journey.enabled? %>
-  <dl class="govuk-summary-list govuk-!-margin-bottom-9">
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:has_case_concluded) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= t(case_details.has_case_concluded, scope: 'values') || t('values.not_asked') %>
-      </dd>
-    </div>
-
-    <% if case_details.case_concluded? %>
+    <% if FeatureFlags.means_journey.enabled? %>
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          <%= label_text(:date_case_concluded) %>
+          <%= label_text(:has_case_concluded) %>
         </dt>
         <dd class="govuk-summary-list__value">
-          <%= l(case_details.date_case_concluded, format: :compact) %>
+          <%= t(case_details.has_case_concluded, scope: 'values') || t('values.not_asked') %>
         </dd>
       </div>
-    <%end%>
 
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:is_preorder_work_claimed) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= t(case_details.is_preorder_work_claimed, scope: 'values') || t('values.not_asked') %>
-      </dd>
-    </div>
+      <% if case_details.case_concluded? %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:date_case_concluded) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= l(case_details.date_case_concluded, format: :compact) %>
+          </dd>
+        </div>
+      <%end%>
 
-    <% if case_details.preorder_work_claimed? %>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          <%= label_text(:preorder_work_date) %>
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= l(case_details.preorder_work_date, format: :compact) %>
-        </dd>
-      </div>
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          <%= label_text(:preorder_work_details) %>
+          <%= label_text(:is_preorder_work_claimed) %>
         </dt>
         <dd class="govuk-summary-list__value">
-          <%= case_details.preorder_work_details %>
+          <%= t(case_details.is_preorder_work_claimed, scope: 'values') || t('values.not_asked') %>
```

Comment:
Same with this one, so it might need a conditional to hide the section if it's not asked `<% if case_details.preorder_work_claimed? %>`